content/docs/user-guide/components/reference/gradients/gradient-baker.md

```diff
@@ -0,0 +1,58 @@
+---
+linktitle: Gradient Baker
+title: Gradient Baker Component
+description: Use the Gradient Baker component to bake a complex node graph of gradients down to a single static image in Open 3D Engine (O3DE).
+---
+
+Use the **Gradient Baker** component to bake a complex node graph of gradients down to a single static image. You can then use a single [Image Gradient](/docs/user-guide/components/reference/gradients/image-gradient/) at runtime, instead of having the entire node graph computed every time, to optimize performance.
+
+For example, take this complex gradient chain with a generator and multiple gradient modifiers.
+
+![Complex chain without gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-chain-without-baker.png)
+
+You can create a **Gradient Baker** and connect its input gradient to the output of the gradient chain you'd like to bake. The **Gradient Baker** also needs an input bounds for where to sample the data. With both of those connected, the graph would look like this.
+
+![Complex chain with gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-chain-with-baker.png)
+
+Clicking the **Bake image** button on the **Gradient Baker** component will then bake out a single static image that can be used in a single [Image Gradient](/docs/user-guide/components/reference/gradients/image-gradient/), which reduces the entire complex graph to only a single component.
+
+![Complex chain with gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-single-image-gradient.png)
+
+The final optimization step is to select all of the gradients in your chain and mark them as **Editor only**. This will prevent them from being loaded/computed at runtime. You can optionally mark the **Gradient Baker** as **Editor only** as well, but it will have no effect because there is no runtime component for it anyways.
+
+![Mark gradient/gradient generators Editor only to disable at runtime](/images/user-guide/components/reference/gradients/gradient-baker-editor-only.png)
+
+## Provider
+
+[Gradient Signal Gem](/docs/user-guide/gems/reference/utility/gradient-signal)
+
+## Gradient Baker properties
+
+![Gradient Baker component properties](/images/user-guide/components/reference/gradients/gradient-baker-component.png)
+
```

Comment:
I know the docs team in general doesn't like pictures of anything that might require maintenance over time. The component card image should get removed, but I'm less sure about the other pictures, since those are helping to illustrate the concept and workflow.

Reply:
Good call. I've removed the image of the component card.

@sptramer @micronAMZN @chanmosq What do y'all think about the other images? They are useful in this case to explain the workflow of why you would use the gradient baker: to be able to condense a complex chain of N gradients/gradient modifiers into a single static image, and then use that single image in an Image Gradient instead.

Reply:
The component card images do tend to go stale faster than other media, but we've been including them in the component reference updates.  I'll ask the team for some guidance on the question.

Reply:
Hey @cgalvan, I think images of component cards can be useful for explaining a workflow, like you do here. 

One thing to keep in mind is that a single image packs in a lot of visual information for the user already. Plus the descriptions/instructions that accompany it, sometimes that's enough for users to understand what's going on.

In this case, you can probably remove the first image (showing the complex gradient chain). The second image alone is a suitable illustration for both paragraphs since it shows the complex gradient chain, as well as the gradient baker, and the accompanying descriptions/instructions clarifies the context for the user.

Reply:
Thanks! I will follow your suggestion as outlined further below (removing the first image and combining its information with the second).